locales/en/messages.json

```
@@ -52,6 +52,12 @@
     "portsSelectManual": {
         "message": "Manual Selection"
     },
+    "portsSelectVirtual": {
+        "message": "Virtual Mode (Experimental)"
```

If the string is not self explanatory, is a good idea to add a `description` text to help the translator do their job. I recommend do that always, some translators have more knowledge and other less of the new features.

```suggestion
        "message": "Virtual Mode (Experimental)",
        "description": "Enables a virtual FC to let navigate by the Configurator without the need of an FC"
```